scoresheets/EGPSR.tex

```diff
@@ -2,14 +2,12 @@
 
 \begin{scorelist}
 	\scoreheading{Main Goal}
-	\scoreitem[3]{800}{Executing the task associated with each command}
+	\scoreitem[3]{150}{Find and clearly state an encountered problem}
```

Comment:
One thing I foresee teams doing is approaching any and all people in the arena and asking them for tasks.
We could say a team wasting their own time is penalty enough.

Reply:
Yes, I think in this case that is penalty enough. Also if the robot repeatedly talks to people without tasks or maybe the same person even they can be removed according to 3.7.13 3. Inactivity

Reply:
Maybe to elaborate. Solving this by giving a penalty for approaching people without a task discourages approaching people in general

Reply:
True, lets see how this plays out at a tournament